src/ui/Button/Button.stories.js

```diff
@@ -0,0 +1,13 @@
+import Button from "./Button.svelte";
+
+export default {
+  title: "Example/Button",
```

Comment:
I assume this should be something else eventually?

Reply:
Or does it not matter?

Reply:
Yeah, it can be something else, it's to add a category in storybook:

<img width="1299" alt="Screen Shot 2020-09-11 at 8 15 15 PM" src="https://user-images.githubusercontent.com/1303908/92982306-9cf1ed00-f46b-11ea-9d10-4171b808f495.png">

https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy

We can change it to be "Components" later on / add subcategories :)